docs/api.md

```diff
@@ -366,6 +366,13 @@ connections, nor does it pause the streams of existing connections or their wire
 
 Resume connecting to new peers.
 
+## `torrent.verifyPieces([function callback (err) {}])`
```

Comment:
The method is supposed to be called `rescanFiles`, right?

Reply:
D'oh! yes.